variables.tf

```diff
@@ -174,3 +174,25 @@ variable "manage_default_security_group" {
   type        = bool
   default     = true
 }
+
+variable "default_security_group_ingress" {
+  description = "List of maps of ingress rules to set on the default security group"
+  type        = list(map(string))
+  default = [{
+    protocol  = -1
+    self      = true
+    from_port = 0
+    to_port   = 0
+  }]
+}
+
+variable "default_security_group_egress" {
+  description = "List of maps of egress rules to set on the default security group"
+  type        = list(map(string))
+  default = [{
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = "0.0.0.0/0"
```

Comment:
It would be better if we replace it with `self = true` - in this case default value would be compatible with those security checks